musicbot/audiocontroller.py

```diff
@@ -327,6 +334,7 @@ async def timeout_handler(self):
         await self.udisconnect()
 
     async def uconnect(self, ctx):
+        self.ctx = ctx
```

Comment:
When the setting `start_voice_channel`  is set it bypasses uconnect() which will have `ctx = none`. See line 71 and 79 in `run.py`. For this feature to work when using these settings uconnect() would need to be reworked to accept an optional voice_channel argument for use in `run.py` and for line 71 & 79 to use uconnect().